.github/workflows/ci.yml

```diff
@@ -282,10 +282,15 @@ jobs:
       matrix:
         target:
           - runner-type: gpu
-            iree-ctest-label-regex: ^requires-gpu|^driver=vulkan$|^driver=cuda$
+            iree-ctest-extra-args:
+              --label-regex ^requires-gpu|^driver=vulkan$|^driver=cuda$
+            iree-nvidia-sm80-tests-disable: 1
```

Comment:
Looks like this should also filter `device-count=2`, since the `test_gpu` job ran `iree/tests/e2e/collectives/collectives_test_multi_gpu`

https://github.com/openxla/iree/actions/runs/5626159982/job/15295889995?pr=14418#step:7:995

Reply:
Done.